tensorflow_datasets/core/lazy_imports_lib_test.py

```diff
@@ -58,6 +60,27 @@ def test_bad_import(self):
     with self.assertRaisesWithPredicateMatch(ImportError, "extras_require"):
       _ = tfds.core.lazy_imports.test_foo
 
+  # pylint: disable=import-outside-toplevel
+  def test_lazy_import_context_manager(self):
+    with tfds.core.lazy_imports():
+      import pandas # pylint: disable=unused-import
+      import matplotlib # pylint: disable=unused-import
+
+    self.assertIn("pandas", sys.modules)
+    self.assertIn("matplotlib", sys.modules)
```

Comment:
If you want to test the module was loaded properly, you should test something like `hasattr(matplotlib, 'function_name')`